### ISSUE TITLE
ImportExportTicket proposes default values and possible columns for de-activated (not activated) fields

### ISSUE THREAD

**Issue by StefanAbel-OTOBO** [RotherOSS/ImportExportTicket#1]
When you add a new Import/Export for tickets, it would be great if only activated options are seen.
E.g. "Responsible" should not be shown at all, when the System Configuration setting "Ticket::Responsible" is activated.

Ticket::Responsible is not activated:
![image](https://github.com/user-attachments/assets/390c3ac4-c101-4f29-a8db-0822ef9dbca6)

In Step 2 you can see the field "Default responsible"
![image](https://github.com/user-attachments/assets/00f1f3a9-678c-4089-adec-7597407b34f5)

In Step 4 you have the possibility to select the responsible or its ID as a possible column:
![image](https://github.com/user-attachments/assets/77e019bb-9c10-40a3-8755-7e06257f31d6)

Also it would be great in Step 2, if the article options are only appear if the checkbox "Import/Export articles" is checked (and maybe a new order of the fields would make sense for a better understanding)

**Comment by bschmalhofer** [comment]
Yes, that makes perfect sense.

**Comment by bschmalhofer** [comment]
@StefanAbel-OTOBO 
I added the check for Ticket::Responsible.  The enhancement in Step 2 is probably best done in JavaScript, which is a bit beyond me.

**Comment by bschmalhofer** [comment]
I reordered the options in step 2 a little bit. The article related settings are now at the end of the list.

**Comment by bschmalhofer** [comment]
At a glance, it looks like there are still things to do for this issue.

**Comment by stefanhaerter** [comment]
@bschmalhofer Can you summarize the remaining ToDos briefly? If time allows, I might have a look.
